src/main/java/net/mcreator/blockly/BlocklyToCode.java

```diff
@@ -173,20 +179,25 @@ public final void processOutputBlock(Element condition) throws TemplateGenerator
 		Element block = conditionBlocks.get(0);
 		String type = block.getAttribute("type");
 
-		boolean generated = false;
-		for (IBlockGenerator generator : blockGenerators) {
-			if (generator.getBlockType() == IBlockGenerator.BlockType.OUTPUT && Arrays.asList(
-					generator.getSupportedBlocks()).contains(type)) {
-				generator.generateBlock(this, block);
+		if (block.getAttribute("disabled").equals("true")) { // Add compile error if block is disabled
+			addCompileNote(new BlocklyCompileNote(BlocklyCompileNote.Type.ERROR,
+					L10N.t("blockly.warnings.disabled_block_type.remove", type)));

```

Comment:
This key should be named `"blockly.errors.disabled_block_type.remove"` as you're using it for a compile _error_ text.